packages/lib/CalendarManager.ts

```diff
@@ -245,12 +245,11 @@ export const getBusyCalendarTimes = async (
   }
 
   // const months = getMonths(dateFrom, dateTo);
+  // Subtract 11 hours from the start date to avoid problems in UTC- time zones.
```

Comment:
Move the `startDate` and `endDate` out of the `try` block so it can be accessed inside of the `catch` block